src/frontend/src/modules/Transactions/FilterTransaction.vue

```diff
@@ -322,6 +340,71 @@ export default {
         this.filter.to = toDate.toISOString().split("T")[0] + " 23:59:59"
       }
     },
+    async loadMiniGrids() {
+      try {
+        await this.miniGridService.getMiniGrids()
+      } catch (error) {
+        console.error("Failed to load mini grids:", error)
+      }
+    },
+
+    async loadCities() {
+      try {
+        await this.cityService.getCities()
+      } catch (error) {
+        console.error("Failed to load cities:", error)
+      }
+    },
+
+    async loadCurrencyList() {
+      try {
+        await this.currencyListService.getCurrencyList()
+      } catch (error) {
+        console.error("Failed to load currency list:", error)
+      }
+    },
+
+    async exportCustomers() {
+      try {
+        const data = {
+          format: this.exportFilters.format,
+        }
+
+        // Add optional filters if they are set
+        if (this.exportFilters.isActive !== null) {
+          data.isActive = this.exportFilters.isActive
+        }
+        if (this.exportFilters.miniGrid) {
+          data.miniGrid = this.exportFilters.miniGrid
+        }
+        if (this.exportFilters.village) {
+          data.village = this.exportFilters.village
+        }
+        if (this.exportFilters.deviceType) {
+          data.deviceType = this.exportFilters.deviceType
```

Comment:
How we using this anywhere? Doesn't seem to be reference anywhere.

Reply:
> Also, I need to know what value "miniGrid" and "village" mean here? Are they ids?

Yes, miniGrid` and `village` should be IDs, I guess